symphony-ws/src/main/java/se/havochvatten/symphony/service/ReportService.java

```diff
@@ -420,4 +415,290 @@ static void writeComparisonRows(Locale locale, SymphonyBandDto[] meta, int[] fea
+            nextCell = totalSheet.getCellByPosition(2, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 1);
+            nextCell.setStringValue("TOTAL");
+            nextCell.getOdfElement().setStyleName("totalHP");
+
+            nextCell = totalSheet.getCellByPosition(2, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 3);
+            nextCell.setStringValue("Ecosystem");
+            nextCell.getOdfElement().setStyleName("pressureHeader");
+            nextCell = totalSheet.getCellByPosition(2, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 4);
+            nextCell.setStringValue("TOTAL");
+            nextCell.getOdfElement().setStyleName("totalHP");
+
+
+            for(p = 0; p < layersToList.pressures.length; p++) {
+                nextCell = totalSheet.getCellByPosition(3 + p, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL);
+                nextCell.setStringValue(pressureTitles.get(cmp.includedPressures[layersToList.pressures[p]]));
+                nextCell.getOdfElement().setStyleName("pressureHeader");
+                nextCell = totalSheet.getCellByPosition(3 + p, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 1);
+                nextCell.setDoubleValue(cmp.totalPerPressure.get(cmp.includedPressures[layersToList.pressures[p]]));
+                nextCell.getOdfElement().setStyleName("totalP");
+                totalSheet.getColumnByIndex(3 + p).setWidth(ODF_TITLE_COLWIDTH);
+            }
+
+            for(e = 0; e < layersToList.ecosystems.length; e++) {
+                nextCell = totalSheet.getCellByPosition(3 + e, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 3);
+                nextCell.setStringValue(ecoTitles.get(cmp.includedEcosystems[layersToList.ecosystems[e]]));
+                nextCell.getOdfElement().setStyleName("pressureHeader");
+                nextCell = totalSheet.getCellByPosition(3 + e, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 4);
+                nextCell.setDoubleValue(cmp.totalPerEcosystem.get(cmp.includedEcosystems[layersToList.ecosystems[e]]));
+                nextCell.getOdfElement().setStyleName("totalP");
+            }
+
+            if(cmpIndex < cmpResults.length - 1) {
+                j = 0;
+                while (j < Math.max(layersToList.pressures.length, layersToList.ecosystems.length) + 2) {
+                    nextCell = totalSheet.getCellByPosition(1 + j, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 6);
+                    nextCell.getOdfElement().setStyleName("resultSep");
+                    ++j;
+                }
+            }
+
+            ++cmpIndex;
+        }
+
+        // Remove the empty sheet from the template document
+        templateDocument.getSpreadsheetTables().get(0).remove();
+
+        List<OdfTable> sheets = templateDocument.getSpreadsheetTables();
+
+        // The "Automatic styles" generated by newTable() factory method apparently causes a
+        // compatibility issue with MS Excel.
+        // Using 'default' table and column styles "ta1" and "co1" effectively remedies this.
```

Comment:
This (admittedly "hacky") fix for the issue is unsufficient.
The mentioned issue will only present a dialog box complaining about "problems with certain content".
However all content, even styling, is preserved after "clicking through". I'm looking into this anyway, meaning there may come a final amendment to the PR to address this.

Reply:
I'm removing this. Contratry to what's claimed in the comment, its ineffective.
It'll be adressed separate from this PR.